spec_opts_unix.go

```diff
@@ -322,7 +323,11 @@ func WithUserID(uid uint32) SpecOpts {
 			}
 		}
 		defer unix.Unmount(root, 0)
-		f, err := os.Open(filepath.Join(root, "/etc/passwd"))
+		ppath, err := fs.RootPath(root, "/etc/passwd")
```

Comment:
Info Req:  why do we have to start from /etc/passwd .  Can we not start from "/"  or container Image location ?